platform/Platform.js

```diff
@@ -109,6 +109,38 @@ class Platform {
     return changed;
   }
 
+  async reloadKernelModule(koName,srcDir,forceReload=false) {
+    const srcPath = `${srcDir}/${koName}.conf`;
+    const dstPath = `/etc/modprobe.d/${koName}.conf`;
+    let koReloaded = false;
+    let confChanged = false;
+    try {
+      await exec(`cmp -s ${srcPath} ${dstPath}`);
+      log.warn(`kernel module ${koName} reload - bypassed due to configuration already up-to-date in ${dstPath}`)
```

Comment:
`preparePlugin()` might be called quite frequently, no point to warn on this. log error and action done only, use `log.verbose()` or `log.debug()` for the rest

Reply:
Sure, let me change the log levels.